scout/adapter/mongo/variant.py

```diff
@@ -713,23 +713,73 @@ def overlapping(self, variant_obj):
 
         return variants
 
-    def evaluated_variants(self, case_id):
+    def evaluated_variant_ids_from_events(self, case_id, institute_id):
+        """Returns variant ids for variants that have been evaluated
+           Return all variants, snvs/indels and svs from case case_id
+            which have an event entry for 'acmg_classification', 'manual_rank', 'dismiss_variant',
+            'cancer_tier', 'mosaic_tags'.
+        Args:
+            case_id(str)
+
+        Returns:
+            variants(iterable(Variant))
+        """
+
+        evaluation_verbs = [
+            "acmg",
+            "manual_rank",
+            "cancer_tier",
+            "dismiss_variant",
+            "mosaic_tags",
```

Comment:
Yeah, I did at first: the issue was with how they are stored and "unstored". But let me take a look again.

Reply:
(and L737 already has `{category: variant}`). 😉

Reply:
Yes, the problem is that we dont really add any "is_commented" to the db store, only to live variant objects. This could be seen as an omission to fix, but it would a) be a rather large operation to fix for old cases before we could rely on it and b) is a bit open to interpretation with regards to global comments c) the whole concept of storing extra keys on variants is scary - I would lean towards implementing an "uncomment" verb event and subtract those much like we do for dismiss variant.

I would suggest keeping the two separated, possibly opening an issue for the missing key on the variant objects.

Reply:
> I would suggest keeping the two separated, possibly opening an issue for the missing key on the variant objects.

Agreed! Then this one is fine to merge I think!